compiler/lib/reserved.ml

```diff
@@ -140,7 +140,25 @@ let provided =
     ; "isNaN"
     ; "parseFloat"
     ; "parseInt"
-    ; "module"
-    ; "require"
+    ; "module" (* only available in node *)
+    ; "require" (* only available in node *)
     ; "Symbol"
+    ; "ArrayBuffer"
+    ; "Float32Array"
+    ; "Float64Array"
+    ; "Int16Array"
+    ; "Int32Array"
+    ; "Int8Array"
+    ; "TextDecoder"
+    ; "Uint16Array"
+    ; "Uint32Array"
+    ; "Uint8Array"
+    ; "atob"
+    ; "btoa"
+    ; "clearInterval"
+    ; "console"
+    ; "global" (* only available in node *)
+    ; "importScripts" (* only available in WebWorker *)
+    ; "performance"
```

Comment:
This is not available on node. Add a comment ?

Reply:
```suggestion
    ; "performance" (* Not available in Node until node 16+ *)
```
This was actually added to node's global scope in node 16.